### PR TITLE
[WIP] Add initial circular-progress component

### PR DESCRIPTION
**Pull request by FWirtz**
Add component for visualization of percentage with an arc.
Add support for custom colors and data.

### CODE REVIEW

**Comment by FWirtz** [comment]
:-1:

File: demo/circular_stacked_demo.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        }
+    </style>
+</head>
+
+<body>
+    <div>
+        <h3>circular-stacked demo (custom element)</h3>
+        <demo-snippet>
+            <template>
+              <circular-stacked primarycolor="#f2503f" accentcolor="#404F70" data='[42, 50]'></circular-stacked>
```
Comment (FWirtz) [inline demo/circular_stacked_demo.html]: accentcolor is deprecated/removed

**Comment by FWirtz** [comment]
:+1:

**Comment by MaximilianV** [comment]
:+1: both diagrams work for me. Even though I'm not an D3-expert :)
Maybe @jak-ing can have a look on it, too?
Anyway, thanks for adding this element @FWirtz!

**Pull request merged**